monkey/tests/unit_tests/monkey_island/cc/services/authentication_service/test_otp_generator.py

```diff
@@ -0,0 +1,15 @@
+from unittest.mock import MagicMock
+
+from monkey_island.cc.services.authentication_service.authentication_facade import (
+    AuthenticationFacade,
+)
+from monkey_island.cc.services.authentication_service.otp_generator import OTPGenerator
+
+
+def test_otp_generator__generates_otp():
```

Comment:
Arguably not even worth the test. Your call.

Reply:
I think although it is not worth it at all, in the coverage it will say not tested and it bothers me 😄